src/modules/metrics/actions.js

```diff
@@ -22,8 +22,8 @@ export function addMetric(item) {
 
 //spaceId must be done before the metric is removed here.
 export function removeMetrics(ids) {
-  if (ids.length === 0) { return }
   return (dispatch, getState) => {
+    if (ids.length === 0) { return }
```

Comment:
There was a redux error shown in the console when an empty cell was used for deletion